web_select_all_companies/__manifest__.py

```diff
@@ -1,18 +1,18 @@
 # Copyright 2023 Camptocamp SA - Telmo Santos
+# Copyright 2024 Alitec Pte Ltd - Jay Patel
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 {
     "name": "Web Select All Companies",
     "summary": "Allows you to select all companies in one click.",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.1",
```

Comment:
version should be 17.0.1.0.0

Reply:
@CRogos Version updated and fixed the commit.